### PR TITLE
Adding a file for Sao Paulo Brazil

### LINKED ISSUE #22: Hometown for douglasreis
## Add file for Sao Paulo, Brazil

- [ ] Create Branch
- [ ] Add a file for Sao Paulo
- [ ] Commit my file
- [ ] Open a Pull Request
- [ ] Have a discussion
- [ ] Merge my Pull Request

### PR DESCRIPTION
**Pull request by douglasreis**
This PR closes #22 

Sup @jsaonlineuk and @irmbrady? :baby_bottle:

### CODE REVIEW

File: places/sao-paulo-brazil.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+## Sao Paulo - Brazil
+
+### Places to Eat
+- Sujinho
+- Bar do Sasha
+- Jardineira
+
+### Things to do
+- Night out at Vila Madalena
+- Atend a football match at Morumbi stadium
```
Comment (douglasreis) [inline places/sao-paulo-brazil.md]: Need to fix speeling

**Comment by irmbrady** [comment]
:+1:

File: places/sao-paulo-brazil.md (hunk 1 above)
Comment (jsaonlineuk) [inline places/sao-paulo-brazil.md]: spppppppppppppppplllllling

**Comment by pauliehyland** [comment]
What's the weather like in Brazil this time of year?

**Pull request merged**